lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -412,6 +412,8 @@ class TypeAnalyzer : public ForwardDataFlowAnalysis<ValueKnowledge> {
       return visitAtenMatmulOp(matmul, operands);
     } else if (auto softmaxIntOp = dyn_cast<AtenSoftmaxIntOp>(op)) {
       return visitAtenSoftmaxIntOp(softmaxIntOp, operands);
+    } else if (auto nts = dyn_cast<PrimNumToTensorScalarOp>(op)) {
```

Comment:
NIT: I think we should follow the naming convention, i.e., name it as `numToTensorOp` rather than nts.

Reply:
This comment is not addressed yet.